api/src/main/java/org/pmiops/workbench/config/WorkbenchConfig.java

```diff
@@ -70,10 +72,23 @@ public static class BillingConfig {
     // The free tier GCP billing account ID to associate with Terra / GCP projects.
     public String accountId;
 
+    // The legacy free tier billing account id that is migrating away. This value helps to make
+    // migration process smooth.
+    public String legacyAccountId;
+
     public String freeTierBillingAccountName() {
       return "billingAccounts/" + accountId;
     }
 
+    public String legacyFreeTierBillingAccountName() {
+      return "billingAccounts/" + legacyAccountId;
+    }
+
+    /// All valid free tier billing accounts, including accountId and legacyAccountId.
+    public Set<String> freeTierBillingAccountNames() {
+      return ImmutableSet.of("billingAccounts/" + accountId, "billingAccounts/" + legacyAccountId);
```

Comment:
nit: probably better that this calls the helpers `of(freetierBilingAccountName(), legacyFreeTierBillingAccountName())`